app/src/main/res/layout/view_description_edit_revert_help.xml

```diff
@@ -5,7 +5,7 @@
     android:layout_height="wrap_content">
 
     <TextView
-        android:id="@+id/view_description_edit_revert_help_contents"
```

Comment:
Why is this string changed @cooltey ..

Reply:
Just want to trim the string length so that it won't be too long in the code. (`viewDescriptionEditRevertHelpContents` => `helpText`).

Reply:
oh sorry my bad... i thought this was a string resource that was changed... sounds good!